### PR TITLE
Feature/expected decimals and ledger

### PR DESCRIPTION
**Pull request by janaakhterov**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Sean-Tedrow-LB — APPROVED** [review]

**Review by andrix10 — APPROVED** [review]

File: reference/cryptocurrency/TransferTransaction.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+---
+
+##### `addTokenTransferWithDecimals` ( `tokenId`: [`TokenId`](reference/token/TokenId.md), `accountId`: `[`AccountId`](reference/cryptocurrency/AccountId.md), `amount`: `Uint64`, `decimals`: `Uint32` ) ): `TransferTransaction`
+
+Add a fungible token transfer with an expected decimals field
+
+---
+
+##### `getTokenIdDecimals`: `Map` < [`TokenId`](reference/token/TokenId.md), `Uint64` >
```
Comment (andrix10) [inline reference/cryptocurrency/TransferTransaction.md]: Shouldn't this be a map of TokenId to Uint32?

**Review by andrix10 — CHANGES_REQUESTED** [review]

File: reference/cryptocurrency/TransferTransaction.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+##### `addTokenTransferWithDecimals` ( `tokenId`: [`TokenId`](reference/token/TokenId.md), `accountId`: `[`AccountId`](reference/cryptocurrency/AccountId.md), `amount`: `Uint64`, `decimals`: `Uint32` ) ): `TransferTransaction`
+
+Add a fungible token transfer with an expected decimals field
+
+---
+
+##### `getTokenIdDecimals`: `Map` < [`TokenId`](reference/token/TokenId.md), `Uint32` >
+
+The the expected decimals per token ID
```
Comment (Sean-Tedrow-LB) [inline reference/cryptocurrency/TransferTransaction.md]: Only need one "the"

File: reference/cryptocurrency/TransferTransaction.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+---
+
+##### `addTokenTransferWithDecimals` ( `tokenId`: [`TokenId`](reference/token/TokenId.md), `accountId`: `[`AccountId`](reference/cryptocurrency/AccountId.md), `amount`: `Uint64`, `decimals`: `Uint32` ) ): `TransferTransaction`
+
+Add a fungible token transfer with an expected decimals field
+
+---
+
+##### `getTokenIdDecimals`: `Map` < [`TokenId`](reference/token/TokenId.md), `Uint32` >
```
Comment (Sean-Tedrow-LB) [inline reference/cryptocurrency/TransferTransaction.md:95]: This does not match [the implementation I wrote for the Java SDK](https://github.com/hashgraph/hedera-sdk-java/blob/30cb10c1d83cd0af948271d692d9e72db618ad93/sdk/src/main/java/com/hedera/hashgraph/sdk/TransferTransaction.java#L63-L71).  Which one should be changed?

**Review by Sean-Tedrow-LB — CHANGES_REQUESTED** [review]

**Review by andrix10 — APPROVED** [review]

**Review by Sean-Tedrow-LB — APPROVED** [review]

**Review by valtyr-naut — APPROVED** [review]
lgtm

**Pull request merged**